tests/lang.sh

```diff
@@ -10,32 +10,49 @@ nix-instantiate --show-trace --eval -E 'builtins.addErrorContext "Hello" (throw
 set +x
 
 fail=0
+just_failed=0
```

Comment:
Is this variable useful? Looks like you could directly set `failed` to 1 in `safe_expect` rather than indirecting through `just_failed`

Reply:
We use it in the last tests to see if we should check the xml. That is why I had to introduce it.